src/NLog/Targets/EventLogTarget.cs

```diff
@@ -320,11 +330,14 @@ private EventLogEntryType GetEntryType(LogEventInfo logEvent)
             if (EntryType != null)
             {
                 //try parse, if fail,  determine auto
-
                 var value = RenderLogEvent(EntryType, logEvent);
 
                 EventLogEntryType eventLogEntryType;
+#if NETSTANDARD2_0
+                if (Enum.TryParse(value, true, out eventLogEntryType))
```

Comment:
could we move this to the EnumHelpers?

Reply:
Not public

Reply:
the line below there is the EnumHelpers? ;)

Reply:
The NetStandard2.0 assembly is not part of the Nlog.dll, it is a separate package and cannot access internal stuff (unless I open for it like with unit-tests).

Reply:
> cannot access internal stuff (unless I open for it like with unit-tests).

Maybe that isn't a bad idea, what do you think? Never done that with "public" assemblies, but guess it works?

Reply:
I don't like replacing a hack  with another hack. The correct solution would be to make an official API. But I can copy-paste the code for this.

Reply:
OK, please copy-paste then for these cases, thx!

Reply:
Now performed copy-paste